src/components/Pools/AddLiquidity/ConfirmPoolDrawer/index.tsx

```diff
@@ -171,7 +171,7 @@ const ConfirmSwapDrawer: React.FC<Props> = (props) => {
   );
 
   const CardConfirmContent = (
-    <Box display="flex" flexDirection="column" p={10} height="100%">
+    <Box display="flex" flexDirection="column" px={20} pb={20} height="100%">
```

Comment:
I think due to this padding, there is some mis alignment in drawer title and content. please check

![Screenshot 2022-09-05 at 3 35 54 PM](https://user-images.githubusercontent.com/6604146/188424265-aa6baa6d-1c90-43df-a60f-16b00dd6b604.png)

Reply:
This should be fine now, I fixed the conflicts.